### ISSUE TITLE
readable subprocess exceptions

### ISSUE THREAD

**Issue by vaaaaanquish-xx** [vaaaaanquish-xx/xontrib-readable-traceback#8]
i want shorting this...
```
xonsh: To log full traceback to a file set: $XONSH_TRACEBACK_LOGFILE = <filename>
Traceback (most recent call last):
  File "<xonsh-code>", line 14, in run_b
  File "/home/shukawai/.pyenv/versions/3.6.3/lib/python3.6/site-packages/xonsh/proc.py", line 559, in __init__
    **kwargs)
  File "/home/shukawai/.pyenv/versions/3.6.3/lib/python3.6/subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "/home/shukawai/.pyenv/versions/3.6.3/lib/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'imgcat': 'imgcat'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shukawai/.pyenv/versions/3.6.3/lib/python3.6/site-packages/xonsh/proc.py", line 1767, in __init__
    proc = spec.run(pipeline_group=pipeline_group)
  File "/home/shukawai/.pyenv/versions/3.6.3/lib/python3.6/site-packages/xonsh/built_ins.py", line 502, in run
    p = self._run_binary(kwargs)
  File "<xonsh-code>", line 25, in run_b
xonsh.tools.XonshError: xonsh: subprocess mode: command not found: imgcat
Did you mean the following?
    msgcat:  Command (/usr/bin/msgcat)
```

**Comment by vaaaaanquish-xx** [comment]
```
xonsh.tools.XonshError: xonsh: subprocess mode: command not found: imgcat
Did you mean the following?
    msgcat:  Command (/usr/bin/msgcat)
```

**Comment by vaaaaanquish-xx** [comment]
This phenomenon occurs in CentOS.
It is necessary to deal with each OS.

**Comment by vaaaaanquish-xx** [comment]
This problem seems to be solved with xonsh 0.6.9.
close.
